common/script/index.js

```diff
@@ -2281,13 +2281,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var totalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (totalStatPoints >= api.maxLevel) {
```

Comment:
Correct. I'm saying we should add an additional stat points constant. Something like:

```js
// statHelpers.js
export const MAX_STAT_POINTS = MAX_LEVEL;

// index.js
var userTotalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
if (userTotalStatPoints >= MAX_STAT_POINTS) {
```

I think it's more semantically meaningful to check against a max stat points constant than a max level constant.